packages/dotnet-sdk/src/TeamsFx/Credential/TeamsUserCredential.cs

```diff
@@ -235,7 +235,7 @@ private async ValueTask<IJSObjectReference> ImportTeamsSdk(IJSRuntime jsRuntime)
     {
         try
         {
-            await jsRuntime.InvokeVoidAsync("import", "https://statics.teams.cdn.office.net/sdk/v1.10.0/js/MicrosoftTeams.min.js").ConfigureAwait(false);
+            await jsRuntime.InvokeVoidAsync("import", "https://res.cdn.office.net/teams-js/2.0.0/js/MicrosoftTeams.min.js").ConfigureAwait(false);
```

Comment:
Can we use the latest version of teams-js? It's 2.4.2 now.

Reply:
Updated. Thanks!